src/zrc/api/viewsets.py

```diff
@@ -284,8 +304,17 @@ class ZaakObjectViewSet(mixins.CreateModelMixin,
         'create': SCOPE_ZAKEN_CREATE,
     }
 
+    def get_kenmerken(self, data):
+        kenmerken = get_kenmerken_from_model(
+            url=data['zaak'],
+            model=Zaak,
+            topics=settings.NOTIFICATIES_KENMERKEN_NAMES
+        )
+        return kenmerken
+
 
-class ZaakInformatieObjectViewSet(NestedViewSetMixin,
+class ZaakInformatieObjectViewSet(NotificationCreateMixin,
```

Comment:
I think using the ViewSetMixin is the better choice over CreateMixin, to prevent future errors.

```suggestion
class ZaakInformatieObjectViewSet(NotificationViewSetMixin,
```

Reply:
In this case I will add `update()` and `destroy()` actions to viewsets which don't support this actions.